.github/workflows/ci.yml

```
@@ -70,7 +70,15 @@ jobs:
 
     name: ${{ matrix.target.os }}-${{ matrix.target.cpu }}${{ matrix.branch != '' && ' (Nim ' || '' }}${{ matrix.branch-short }}${{ matrix.branch != '' && ')' || '' }}
     runs-on: ${{ matrix.builder }}
+
     steps:
+      - name: Set XDG_CACHE_HOME
+        run: echo "XDG_CACHE_HOME=${{ runner.temp }}/.cache" >> $GITHUB_ENV
```

A better name would help people understand the purpose of this:
```suggestion
      - name: Fix Nim cache conflicts
        run: echo "XDG_CACHE_HOME=${{ runner.temp }}/.cache" >> $GITHUB_ENV
```